acrobat/scripts/scripts.js

```diff
@@ -461,20 +461,20 @@ const { ietf } = getLocale(locales);
   }
 
   setConfig({ ...CONFIG, miloLibs });
+
+  loadIms().then(() => {
+    const imsIsReady = new CustomEvent('IMS:Ready');
+    window.dispatchEvent(imsIsReady);
+  });
+
   loadLana({ clientId: 'dxdc', tags: 'DC_Milo' });
 
-  // get event back form dc web and then load area
```

Comment:
This comment no longer seems to make sense. Any idea what event this referred to?